static/js/translation.js

```diff
@@ -0,0 +1,60 @@
+
+//load initial language file.
+languageStartup();
+
+function languageStartup(){
+    var url = window.location.href;
+    console.log(url);
```

Comment:
Was this left here on purpose?

Reply:
And I thought I've removed all the debug logs, my bad I'll remove it next push